samples/bluetooth/unicast_audio_server/src/main.c

```diff
@@ -18,19 +18,39 @@
 
 #define MAX_PAC 1
 
+#define AVAILABLE_SINK_CONTEXT        (BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED | \
+BT_AUDIO_CONTEXT_TYPE_CONVERSATIONAL | BT_AUDIO_CONTEXT_TYPE_MEDIA | \
+BT_AUDIO_CONTEXT_TYPE_GAME | BT_AUDIO_CONTEXT_TYPE_INSTRUCTIONAL)
+#define AVAILABLE_SOURCE_CONTEXT      (BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED | \
+BT_AUDIO_CONTEXT_TYPE_CONVERSATIONAL | BT_AUDIO_CONTEXT_TYPE_MEDIA | \
+BT_AUDIO_CONTEXT_TYPE_GAME)
+
 /* Mandatory support preset by both client and server */
 static struct bt_audio_lc3_preset preset_16_2_1 = BT_AUDIO_LC3_UNICAST_PRESET_16_2_1;
 
 NET_BUF_POOL_FIXED_DEFINE(tx_pool, 1, CONFIG_BT_ISO_TX_MTU, NULL);
 static struct bt_conn *default_conn;
 static struct bt_audio_stream streams[MAX_PAC];
 
+
+static uint8_t unicast_server_addata[] = {
+	BT_UUID_16_ENCODE(BT_UUID_ASCS_VAL), /* ASCS UUID */
+	0x01, /* Target Announcement */
+	(((AVAILABLE_SINK_CONTEXT) >>  0) & 0xFF),
+	(((AVAILABLE_SINK_CONTEXT) >>  8) & 0xFF),
+	(((AVAILABLE_SOURCE_CONTEXT) >>  0) & 0xFF),
+	(((AVAILABLE_SOURCE_CONTEXT) >>  8) & 0xFF),
+	0x00, /* Metadata length */
+};
+
 /* TODO: Expand with BAP data */
 static const struct bt_data ad[] = {
 	BT_DATA_BYTES(BT_DATA_FLAGS, (BT_LE_AD_GENERAL | BT_LE_AD_NO_BREDR)),
 	BT_DATA_BYTES(BT_DATA_UUID16_ALL, BT_UUID_16_ENCODE(BT_UUID_ASCS_VAL)),
+	BT_DATA(BT_DATA_SVC_DATA16, unicast_server_addata, ARRAY_SIZE(unicast_server_addata)),
```

Comment:
I think you can also do something like 
```
BT_DATA_BYTES(BT_DATA_SVC_DATA16, unicast_server_addata)
```

Reply:
> BT_DATA_SVC_DATA16

Just tried, I think BT_DATA_BYTES is good at for byte-wise operation, BT_DATA is for array-wise. Let's still keep BT_DATA.

Reply:
`BT_DATA_BYTES` is basically just adding the size of the content from `BT_DATA`. 

Normally you'd also use `sizeof` instead of `ARRAY_SIZE` for `BT_DATA`. It works right now since `unicast_server_addata` is a `uint8_t`, so `ARRAY_SIZE` returns the same value as `sizeof`. 

Its OK to keep, but `BT_DATA_BYTES` would just make this line slightly more simple.

Reply:
> `BT_DATA_BYTES` is basically just adding the size of the content from `BT_DATA`.
> 
> Normally you'd also use `sizeof` instead of `ARRAY_SIZE` for `BT_DATA`. It works right now since `unicast_server_addata` is a `uint8_t`, so `ARRAY_SIZE` returns the same value as `sizeof`.
> 
> Its OK to keep, but `BT_DATA_BYTES` would just make this line slightly more simple.

I see, I tried some ways to test and the issue might be this:
below is BT_DATA_BYTES definition, you can see that it's friendly if _bytes are constant value, for example (0x12, 0x34, 0x56) and (uint8_t []) make it as an array type. I saw some declarations in different samples. 
However, if I put unicast_server_addata (an array name) instead of _bytes, it will cause a problem because ((uint8_t []) { unicast_server_addata } is incorrect. 

**#define BT_DATA_BYTES(_type, _bytes...) \
	BT_DATA(_type, ((uint8_t []) { _bytes }), \
		sizeof((uint8_t []) { _bytes }))**

Reply:
I see, so `BT_DATA_BYTES` doesn't work on variables, but only array literals? 

Fair enough, then `BT_DATA` works :)